webapp/exercise_admin/templates/exercise_admin/base.html

```diff
@@ -216,9 +223,8 @@ <h2 id="edit-feedback-div-title">Edit feedback question:</h2>
           <ul class="exercise-tags">
             {% for ex_tag in exercise.tags %}
               <li class="exercise-tag">
-                <input type="text" id="exercise-tag-{{ forloop.counter0|add:"1" }}" class="exercise-tag-input"
-                       name="exercise_tag_{{ forloop.counter0|add:"1" }}" value="{{ ex_tag }}" maxlength="32"
-                       onfocus="highlight_parent_li(this);" oninput="change_tag_width(this);">
+                <input type="text" id="exercise-tag-{{ forloop.counter }}" class="exercise-tag-input" name="exercise_tag_{{ forloop.counter }}"
+                       value="{{ ex_tag }}" maxlength="32" onfocus="highlight_parent_li(this);" oninput="change_tag_width(this);">
```

Comment:
Revert back to three line split of attributes (see pull request diff for details). The lines are too long this way.